lib/kb_uploadmethods/Utils/UploaderUtil.py

```diff
@@ -330,3 +330,27 @@ def _upload_file_url(self, params):
         result = ru.upload_reads(upload_file_params)
 
         return result
+
+    def update_staging_service(self, staged_file, obj_ref):
```

Comment:
I would add a test to see if this really updates UPA in file metadata.

Reply:
The updating of UPA is done in the staging_service, by sending a post. The response from the post indicates whether it was successful or not, which is checked and an error is returned on failure.

Reply:
👍